### PR TITLE
Add option to correctly parse log

### LINKED ISSUE #10: [BUG] Unexpected encoding in url.path for request completed
## Describe the bug

Unexpected `&` encoing for path query parameters

The `request completed` log presents encoded `&` when logging `url.path` key, below follows an example log from a service

```
{"host":{"hostname":"myservice","forwardedHost":"forwardedhost"},"http":{"request":{"method":"GET","userAgent":{"original":"got/9.6.0 (https://github.com/sindresorhus/got)"}},"response":{"statusCode":200,"body":{"bytes":2}}},"level":30,"msg":"request completed","reqId":"xyz","responseTime":78,"time":1608556458,"url":{"path":"/users?_l=50\u0026_sk=0\u0026username=fe*\u0026email=fe*"}}
```

## How To Reproduce

Invoke a path using query parameters for an API that uses glogger middleware

## Expected behavior

Proper information encoding

## System Info

- Go Version 1.15.2

### PR DESCRIPTION
**Pull request by davidebianchi**
This closes #10 

I've added an option to disable html escape. It is optional, default to false for retro compatibility.

This could also be seen as a library bug, therefore not a breaking changes, but it could bring to security issues if not correctly rendered

### CODE REVIEW

File: jsonformatter.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// JSONFormatter will format logs in JSON with
```
Comment (fredmaggiowski) [inline jsonformatter.go:27]: with what? 🤔 


Maybe better?

```suggestion
// JSONFormatter struct formats logs in JSON following Mia-Platform guidelines.
```

**Review by fredmaggiowski — APPROVED** [review]
I'm not completely convinced that this feature should be disabled by default. I'm ok with the option to allow control over it (since it can bring security issues) but since we're formatting logs I don't see these security concerns as a big issue (we're not rendering HTML during log collection).

Having the flag with default value to `false` means that we'll have to alwasy set it to `true` in our services, while I'd expect this to be the default behaviour (think about the node.js logger etc)

**Comment by davidebianchi** [comment]
Yes, I don't like this as optional. I also suppose this should be the default value, but this could bring to a major update of the lib since it change the default behaviour with possible security issues (we do not render log in html, but maybe someone else do it)

**Review by fredmaggiowski — APPROVED** [review]

**Comment by fredmaggiowski** [comment]
I do not agree that it should be a breaking change to have to option true by default since it would correct a behaviour that we didn't expect in our use case... It's ok to merge it anyway

**Pull request merged**